packages/notebook-extension/src/index.ts

```diff
@@ -2062,6 +2076,12 @@ function populateMenus(
         () => void 0
       );
     },
+    runAllMarkdown: current => {
```

Comment:
This is where you would add a group containing the `renderAllMarkdown` to the "Run" menu, rather than adding it to the `codeRunners`.